### PR TITLE
docs: add getting started and clean up docs

### LINKED ISSUE #2392: docs: getting started
We want to provide a guide for getting started with the web components library.
This should be done in https://github.com/sbb-design-systems/digital.sbb.ch

### PR DESCRIPTION
**Pull request by jeripeierSBB**
Closes #2392

### CODE REVIEW

File: docs/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+- **Be welcoming.** We strive to be a community that welcomes and supports
+  people of all backgrounds and identities.
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our community a harassment-free experience
+for everyone, regardless of age, body size, visible or invisible disability, ethnicity, sex characteristics,
+gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance,
+race, religion, or sexual identity and orientation.
```
Comment (DavideMininni-Fincons) [inline docs/CODE_OF_CONDUCT.md:17]:
```suggestion
race, political and religious beliefs, or sexual identity and orientation.
```

File: docs/GETTING_STARTED.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+2. Including typography is required to apply all SBB styles to your application.
+
+   ```css
+   @import 'node_modules/@sbb-esta/lyne-components/typography.css';
+   ```
+
+3. Import the desired element and add it to globalThis:
+
+   ```ts
+   import { SbbButtonElement } from '@sbb-esta/lyne-components/button';
```
Comment (jeripeierSBB) [inline docs/GETTING_STARTED.md:31]: check other occurrences as well

```suggestion
   import { SbbButtonElement } from '@sbb-esta/lyne-components/button.js';
```

**Review by TomMenga — APPROVED** [review]
Good job 👍 
I've run a correctness check with Grammarly and fixes some typos (here's the [commit](https://github.com/lyne-design-system/lyne-components/commit/2b03ff2a4cff28aed4f11ed361a1960c171d0cb1))

**Review by mcilurzo — APPROVED** [review]

File: docs/GETTING_STARTED.md [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+  <sbb-component class="sbb-enable-animation">
+    <!-- animations will play -->
+  </sbb-component>
+</sbb-component>
+```
+
+### Units
+
+All our tokens and components use `rem` as size unit.
+We strongly recommend using the `rem` unit all over the project to guarantee consistent scaling
```
Comment (kyubisation) [inline docs/GETTING_STARTED.md:282]:
```suggestion
We strongly recommend using the `rem` unit to guarantee consistent scaling
```

File: docs/GETTING_STARTED.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+when font size changes in browsers.
+
+### Stacking
+
+As we can't use popover API so far, stacking of overlay context is done manually.
```
Comment (kyubisation) [inline docs/GETTING_STARTED.md:287]:
```suggestion
As we can't use popover API yet, stacking of overlay context is done manually.
```

File: docs/GETTING_STARTED.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+However, this can interfere with the z-index of your components.
+Therefore, every overlay component provides a CSS variable to override its z-index.
+Additionally, there is the global CSS variable `--sbb-overlay-default-z-index` that has a default z-index of 1000.
+So developers have the chance to either globally or component-specifically change the z-index.
```
Comment (kyubisation) [inline docs/GETTING_STARTED.md:291]:
```suggestion
With this developers have the chance to change the z-index either globally or on component level.
```

File: docs/REVIEW.md [hunk 6]
```diff
@@ -25,10 +25,10 @@ For progress indication, we use labels to indicate a specific expectation:
 
 ## Review Checks
 
-This document shows what should be done during review process.
+This document shows what should be done during the review process.
 
 A basic review only contains the entries which have an exclamation mark (⚠️).
-Depending on circumstances, additional options might be considered.
+Depending on the circumstances, additional options might be considered.
```
Comment (kyubisation) [inline docs/REVIEW.md:31]:
```suggestion
Depending on the circumstance, additional options might be considered.
```

**Review by kyubisation — APPROVED** [review]
LGTM 👍

**Pull request merged**